docs/pages/admin-guides/teleport-policy/policy-how-to-use.mdx

```diff
@@ -57,8 +71,8 @@ Access Graph divides your infrastructure into six main components:
 
 Identities are the actors that can access your infrastructure. They can be employees, contractors, machines or bots.
 
-The number on the right hand side shows "standing privileges". Standing privileges is the number of 
-resources that an identity can access without creating an access request.
+The number on the right hand side shows "standing privileges". Standing privileges is the number of
```

Comment:
"Standing privileges is" doesn't follow conventional English subject-verb agreement. Is there another way to describe this metric? Maybe, "The standing privileges metric indicates...".